src/msg/simple/Accepter.cc

```diff
@@ -327,7 +327,7 @@ void *Accepter::entry()
     if (pfd[1].revents & (POLLIN | POLLERR | POLLNVAL | POLLHUP)) {
       // We got "signaled" to exit the poll
       // clean the selfpipe
-      if (::read(shutdown_rd_fd, &ch, 1) == -1) {
+      if (::read(shutdown_rd_fd, ch, 1) == -1) {
```

Comment:
i don't think it's advisable to write to buffer pointed by uninitialized pointer.

Reply:
@tchaikov Closing this. Since you have already created above PR to address Issue.
Thanks